src/daria/utils/conversions.py

```diff
@@ -46,9 +46,9 @@ def BGR2RED(img: da.Image) -> da.Image:
         da.Image: converged image
     """
     img_red = img.copy()
-    if img.colorspace == "bgr":
+    if img.colorspace == "BGR":
         img_red.img = img_red.img[:, :, 2]
-    elif img.colorspace == "rgb":
+    elif img.colorspace == "RGB":
         img_red.img = img_red.img[:, :, 0]
```

Comment:
Why not using simply: `img_red.toRED()` - this would in particular also change the value of `colorspace`.

Reply:
Yes.. I actually do not really know if we even would like to keep these functions. They are now anyways a part of the Image class, and of someone wants to change the color of something externally perhaps they can just use cv2.

Reply:
In some sense it is natural to request havign such functionality. Note that the function aims at returning a `daria.Image` and not a `np.ndarray`. Thus, cv2 may be not sufficient. Instead of having these functions, one could however consider use a flag in, e.g., `toRED()` controlling whether the current object is altered or another object is created. It could look like: `def toRED(self, return_image: bool = False) -> Optional[daria.Image]`. And if `return_image=True`, a new object is created instead of changing `self`.